lib/index.js

```diff
@@ -1,5 +1,6 @@
 // import 'antd/dist/antd.css';
 import 'antd/dist/antd.less';
+import './components/button/button.css';
```

Comment:
Why are we importing the style at this layer and not at the button layer like the other styles?

Reply:
We don't create a new component we just overwrite some styles for global button's styles

Reply:
Yes, but since all the other functions utilize the `Button` component from Virtuoso, why don't we import it at the Button layer?

How have we handled this in other ways?

Reply:
Sorry, but I don't understand) 
Now we class like `.ant-button` and we need to change the color for `:hover` so I added these changes for global classes, this CSS was added to the compiled file `dist/bundle.cjs.css`.
![image](https://user-images.githubusercontent.com/85224435/171667294-14fcad59-2e7d-4d6d-8be9-10c75e5371f0.png)